app/packages/vhx-style-tables/docs/code.html

```diff
@@ -90,6 +90,38 @@
         </tr>
       </tbody>
     </table>
+
+    <!-- Block -->
+    <table class="table table--block margin-bottom-large">
+      <thead>
+        <th class="text-right" style="width: 20%">ID</th>
```

Comment:
how about for the example, we use our width classes instead of explicit widths with the style attribute? i believe this would still work. 

this way we can adjust as needed for different screen sizes as well. and having it in the example can show our recommended implementation. 

what do you think?

example..
```
<table class="table table--block margin-bottom-large">
  <thead>
    <th class="text-right small-8"></th>
    <th class="text-right small-8"></th>
 </thead>
 <tbody>
    <tr>
      <td class="small-8"></td>
      <td class="small-8"></td>
  </tr>
 </tbody>
</table>
```